src/core/compile.ts

```diff
@@ -15,9 +15,6 @@ export function compile(
     // Make sure to keep the original spec as is
     const _spec = JSON.parse(JSON.stringify(spec));
 
-    // Override default visual encoding (i.e., `DataTrack` => `BasicSingleTrack`)
-    overrideDataTemplates(_spec);
```

Comment:
Removed this since `DataTrack` was deprecated but was not removed from this part.